Resources/Textures/Objects/Specific/Hydroponics/cocoa.rsi/meta.json

```diff
@@ -0,0 +1,44 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from https://github.com/vgstation-coders/vgstation13",
```

Comment:
```suggestion
  "copyright": "Taken from https://github.com/vgstation-coders/vgstation13. produce-beans drawn by FillerVK",
```

Reply:
Okay, if that's all, then please merge my commit